tests/keras_contrib/test_objectives.py

```diff
@@ -25,5 +24,32 @@ def test_objective_shapes_2d():
         assert K.eval(objective_output).shape == (6,)
 
 
+def test_dssim_same():
+    x = np.random.random_sample(30 * 30 * 3).reshape([1, 30, 30, 3])
+    x1 = KC.variable(x)
+    loss = objectives.DSSIMObjective(1)
+    np.allclose([0.0], KC.eval(loss(x1, x1)))
```

Comment:
`np.allclose` won't fail the test if it returns False - you could use the numpy function `from numpy.testing import assert_allclose` or just apply an asser(... == True)